iceoryx_posh/source/runtime/ipc_runtime_interface.cpp

```diff
@@ -0,0 +1,271 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2020 by Apex.AI Inc. All rights reserved.
```

Comment:
wrong year

Reply:
See [here](https://github.com/eclipse-iceoryx/iceoryx/pull/538#discussion_r573644456)

Reply:
The class was called MqRuntimeInterface before, therefore you touched and changed the code!

Reply:
Alrighty, will adapt. In previous pull requests we defined that copyright changes only apply to functional changes. In this case the code was only moved and no behavioural changes was introduced.

@dkroenke Shall we document that somewhere?